modern/src/map/mapStyles.js

```diff
@@ -1,10 +1,11 @@
-export const styleCustom = (url, attribution) => ({
+export const styleCustom = (url, attribution, tileSize) => ({
   version: 8,
   sources: {
     osm: {
       type: 'raster',
       tiles: [url],
       attribution: attribution,
+      tileSize: tileSize,
```

Comment:
I think it's better to just hardcode 256 for now. Is there any place where we use different number?

Reply:
No, a different number is not used. I hardcoded the value as you said.